src/System.Private.CoreLib/Resources/Strings.resx

```diff
@@ -1999,36 +1954,12 @@
   <data name="CancellationToken_CreateLinkedToken_TokensIsEmpty" xml:space="preserve">
     <value>No tokens were supplied.</value>
   </data>
-  <data name="CancellationToken_SourceDisposed" xml:space="preserve">
-    <value>The CancellationTokenSource associated with this CancellationToken has been disposed.</value>
-  </data>
   <data name="CancellationTokenSource_Disposed" xml:space="preserve">
     <value>The CancellationTokenSource has been disposed.</value>
   </data>
   <data name="ConcurrentCollection_SyncRoot_NotSupported" xml:space="preserve">
     <value>The SyncRoot property may not be used for the synchronization of concurrent collections.</value>
   </data>
-  <data name="event_SpinLock_FastPathFailed" xml:space="preserve">
-    <value>SpinLock beginning to spin.</value>
-  </data>
-  <data name="event_SpinWait_NextSpinWillYield" xml:space="preserve">
-    <value>Next spin will yield.</value>
-  </data>
-  <data name="event_TaskCompleted" xml:space="preserve">
-    <value>Task {2} completed.</value>
-  </data>
-  <data name="event_TaskScheduled" xml:space="preserve">
-    <value>Task {2} scheduled to TaskScheduler {0}.</value>
-  </data>
-  <data name="event_TaskStarted" xml:space="preserve">
-    <value>Task {2} executing.</value>
-  </data>
-  <data name="event_TaskWaitBegin" xml:space="preserve">
-    <value>Beginning wait ({3}) on Task {2}.</value>
-  </data>
-  <data name="event_TaskWaitEnd" xml:space="preserve">
-    <value>Ending wait on Task {2}.</value>
-  </data>
```

Comment:
These event_ resources are used EventSource, concatenating "event_" with the event name at run time.